src/main/java/com/rtb/projectmanagementtool/task/TaskController.java

```diff
@@ -0,0 +1,82 @@
+package com.rtb.projectmanagementtool.task;
+
+import java.util.HashSet;
+
+/** Class controlling the TaskData object. */
+public final class TaskController {
+
+  private TaskData taskData;
+
+  public TaskController(TaskData taskData) {
+    this.taskData = taskData;
+  }
+
+  // Is there a way to avoid retyping all of these getters and setters?
```

Comment:
Eclipse has a "generate all getters/setters" functionality.  I'm not sure if the Cloud Editor has something similar (probably not).  But as per comment above, you won't need to duplicate here since field getters/setters would only be needed on the Data object.